shell/tools/find-unused-module.ts

```diff
@@ -38,16 +38,27 @@ function findUnusedModule(options: Options) {
   const allFiles = fastGlob.sync(includeContentPath).map((item) => normalize(item));
   const entryModules: string[] = [];
   const usedModules: string[] = [];
+  let exportModules: string[] = [];
+  let importModules: string[] = [];
 
   setRequirePathResolver(resolveRequirePath);
   entries.forEach((entry) => {
     const entryPath = resolve(cwd, entry);
     entryModules.push(entryPath);
-    traverseModule(entryPath, (modulePath: string) => {
+    traverseModule(entryPath, (modulePath: string, items: string[], isImport: boolean) => {
       usedModules.push(modulePath);
+      if (isImport) {
+        importModules = importModules.concat(...items);
+      } else {
+        exportModules = exportModules.concat(...items);
+      }
     });
   });
 
+  const importSet = new Set(importModules);
+  const exportSet = new Set(exportModules);
+  const unusedExport = new Set([...exportSet].filter((x) => !importSet.has(x)));
```

Comment:
This set is useless ? ExportSet is already an unduplicated set, which means its subset is also an unduplicated array.